pennylane_orquestra/utils.py

```diff
@@ -0,0 +1,195 @@
+            ``wires`` is not available, will be set to 1.
+
+    Returns:
+        Wires: Cleaned wire mapping with indices corresponding to qubits and values
+            corresponding to wire labels.
+
+    **Example**
+
+    >>> # consec int wires if no wires mapping provided, ie. identity map: 0<->0, 1<->1, 2<->2
+    >>> _process_wires(None, 3)
+    <Wires = [0, 1, 2]>
+
+    >>> # List as mapping, qubit indices with wire label values: 0<->w0, 1<->w1, 2<->w2
+    >>> _process_wires(['w0','w1','w2'])
+    <Wires = ['w0', 'w1', 'w2']>
+
+    >>> # Wires as mapping, qubit indices with wire label values: 0<->w0, 1<->w1, 2<->w2
+    >>> _process_wires(Wires(['w0', 'w1', 'w2']))
+    <Wires = ['w0', 'w1', 'w2']>
+
+    >>> # Dict as partial mapping, int qubits keys to wire label values: 0->w0, 1 unchanged, 2->w2
+    >>> _process_wires({0:'w0',2:'w2'})
+    <Wires = ['w0', 1, 'w2']>
+
+    >>> # Dict as mapping, wires label keys to consec int qubit values: w2->2, w0->0, w1->1
+    >>> _process_wires({'w2':2, 'w0':0, 'w1':1})
+    <Wires = ['w0', 'w1', 'w2']>
+    """
+
+    # infer from wires, or assume 1 if wires is not of accepted types.
+    if n_wires is None:
+        n_wires = len(wires) if isinstance(wires, (Wires, list, tuple, dict)) else 1
+
+    # defaults to no mapping.
+    if wires is None:
+        return Wires(range(n_wires))
+
+    if isinstance(wires, (Wires, list, tuple)):
+        # does not care about the tail if more wires are provided than n_wires.
+        wires = Wires(wires[:n_wires])
+
+    elif isinstance(wires, dict):
+
+        if all([isinstance(w, int) for w in wires.keys()]):
+            # Assuming keys are taken from consecutive int wires. Allows for partial mapping.
+            n_wires = max(wires) + 1
+            labels = list(range(n_wires))  # used for completing potential partial mapping.
+            for k, v in wires.items():
+                if k < n_wires:
+                    labels[k] = v
+            wires = Wires(labels)
+        elif set(range(n_wires)).issubset(set(wires.values())):
+            # Assuming values are consecutive int wires (up to n_wires, ignores the rest).
+            # Does NOT allow for partial mapping.
+            wires = {v: k for k, v in wires.items()}  # flip for easy indexing
+            wires = Wires([wires[i] for i in range(n_wires)])
+        else:
+            raise ValueError("Expected only int-keyed or consecutive int-valued dict for `wires`")
+
+    else:
+        raise ValueError(
+            "Expected type Wires, list, tuple, or dict for `wires`, got {}".format(type(wires))
+        )
+
+    if len(wires) != n_wires:
+        # check length consistency when all checking and cleaning are done.
+        raise ValueError(
+            "Length of `wires` ({}) does not match `n_wires` ({})".format(len(wires), n_wires)
+        )
+
+    return wires
+
+
+def _terms_to_qubit_operator_string(coeffs, ops, wires=None):
```

Comment:
This function is an adaptation of [`_terms_to_qubit_operator` in PennyLane-Qchem](https://github.com/PennyLaneAI/pennylane/blob/88be620a04a802380d07dbdd84acb48a8f245fe6/qchem/pennylane_qchem/qchem/structure.py#L552). Instead of creating an `openfermion.QubitOperator` internally, it creates the string representation of the operator.

It was added separately to PennyLane-Orquestra to minimize the number of dependencies of the plugin.